<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
}

QPushButton::pressed {
	background-color:	qlinegradient(x1: 0, x2: 0, y1: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
}

QPushButton::pressed {
	background-color:	qlinegradient(x1: 0, x2: 0, y1: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
}

QPushButton::pressed {
	background-color:	qlinegradient(x1: 0, x2: 0, y1: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_multiply">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>120</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 250, 99);
color: rgb(0, 0, 0);

QPushButton::Pressed {
	
	background-color:	qlinegradient(x1: 0, x2: 0, y1: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>180</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
}

QPushButton::pressed {
	background-color:	qlinegradient(x1: 0, x2: 0, y1: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_subtract">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>180</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 250, 99);
color: rgb(0, 0, 0);

QPushButton::Pressed {
	
	background-color:	qlinegradient(x1: 0, x2: 0, y1: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
}

QPushButton::pressed {
	background-color:	qlinegradient(x1: 0, x2: 0, y1: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
}

QPushButton::pressed {
	background-color:	qlinegradient(x1: 0, x2: 0, y1: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>240</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
}

QPushButton::pressed {
	background-color:	qlinegradient(x1: 0, x2: 0, y1: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_add">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>240</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 250, 99);
color: rgb(0, 0, 0);

QPushButton::Pressed {
	
	background-color:	qlinegradient(x1: 0, x2: 0, y1: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
}

QPushButton::pressed {
	background-color:	qlinegradient(x1: 0, x2: 0, y1: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
}

QPushButton::pressed {
	background-color:	qlinegradient(x1: 0, x2: 0, y1: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dot">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>300</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(0, 232, 170);
	color: rgb(0, 0, 0);
}

QPushButton::pressed {
	
	background-color:	qlinegradient(x1: 0, x2: 0, y1: 0, y2: 1, stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_equal">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>300</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 250, 99);
color: rgb(0, 0, 0);

QPushButton::Pressed {
	
	background-color:	qlinegradient(x1: 0, x2: 0, y1: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>120</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
}

QPushButton::pressed {
	background-color:	qlinegradient(x1: 0, x2: 0, y1: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_invertSign">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(0, 232, 170);
	color: rgb(0, 0, 0);
}

QPushButton::pressed {
	
	background-color:	qlinegradient(x1: 0, x2: 0, y1: 0, y2: 1, stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_divide">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 250, 99);
color: rgb(0, 0, 0);

QPushButton::Pressed {
	
	background-color:	qlinegradient(x1: 0, x2: 0, y1: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(0, 232, 170);
	color: rgb(0, 0, 0);
}

QPushButton::pressed {
	
	background-color:	qlinegradient(x1: 0, x2: 0, y1: 0, y2: 1, stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>CE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_percent">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(0, 232, 170);
	color: rgb(0, 0, 0);
}

QPushButton::pressed {
	
	background-color:	qlinegradient(x1: 0, x2: 0, y1: 0, y2: 1, stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QLabel" name="resultShow">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>240</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font: 16pt &quot;Segoe UI&quot;;
	qproperty-alignment: 'AlignVCenter | AlignRight';
	border: 1px solid gray;
	
	color: rgb(0, 0, 0);
}

background-color: white;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
